<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPS_data</class>
 <widget class="QDialog" name="GPS_data">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-image: url(:/icons/GPS_bgrnd2.jpg);}</string>
  </property>
  <widget class="QLabel" name="lblLatitude_val">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>95</y>
     <width>180</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblAltitude_val">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>188</y>
     <width>180</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblLongitude_val">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>141</y>
     <width>180</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblAltitude">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>188</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Altitude : </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNo_sat_val">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>275</y>
     <width>130</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblGrndSpeed">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>231</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Speed km/h :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblGrndSpeed_val">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>231</y>
     <width>130</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblLatitude">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>95</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Latitude : </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLongitude">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>141</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Longitude : </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNo_sat">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>275</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>No of Satellites : </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLatitude_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GPS Information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(106, 35, 35);border-width: 1px;border-color:rgb(0, 0, 0); border-style: solid;font: 75 12pt &quot;Serif&quot;; 

</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Download">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(106, 35, 35);border-width: 1px;border-color:rgb(0, 0, 0); border-style: solid;font: 75 12pt &quot;Serif&quot;; 

</string>
   </property>
   <property name="text">
    <string>AGPSDownload</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
